regulations/views/comment.py

```diff
@@ -70,3 +71,29 @@ def submit_comment(request):
         'status': 'submitted',
         'metadata_url': metadata_url,
     })
+
+
+@csrf_exempt
+@require_http_methods(['GET', 'HEAD'])
+def get_federal_agencies(request):
+    url = "{}?{}".format(settings.REGS_GOV_API_LOOKUP_URL,
```

Comment:
Aside: I never understood why `urlparse.urljoin` didn't handle this well. When I need to do lots of URL manipulation, I usually pull in a third-party library like `furl` or `purl`.